cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -108,6 +127,12 @@ func NewCmdCreateCR(ioStreams genericclioptions.IOStreams, factory cmdutil.Facto
 		"Path to a file containing a Certificate resource used as a template when generating the CertificateRequest resource")
 	cmd.Flags().StringVar(&o.KeyFilename, "output-key-file", o.KeyFilename,
 		"Name of file that the generated private key will be written to")
+	cmd.Flags().StringVar(&o.CertFileName, "output-cert-file", o.CertFileName,
```

Comment:
For consistency with `--from-certificate-file`, should this be `--output-certificate-file`?